### PR TITLE
Add join table

### PR DESCRIPTION
**Pull request by Mike47ip**
### Project requirements
- Create a table named `vets` with the following columns:
  - `id`: integer (set it as autoincremented PRIMARY KEY)
  - `name`: string
  - `age`: integer
  - `date_of_graduation`: date
- There is a **many-to-many** relationship between the tables `species` and `vets`: a vet can specialize in multiple species, and a species can have multiple vets specialized in it. Create a "join table" called `specializations` to handle this relationship.

_SCREENSHOTS_ 

244427443-c1ddce14-3d98-474f-93bd-884f8cefc745

244427142-c73aa5e0-a5b9-40b4-8a84-b214455714da

244428029-ba2ded52-9e0b-46f8-8ba2-87bb6dd375a1

244427443-c1ddce14-3d98-474f-93bd-884f8cefc745

244428186-d88e084b-13f2-41e2-8285-6f219a1a9ab6

244428404-084b47fa-1960-4130-ac8f-f15125def795

244428808-2c2288b5-430e-49b6-a9bd-183a0afb060c

244428983-f316c834-04f8-4f57-b43d-8bc4415fed02

244429316-8335964e-a561-476a-8b8e-10cb4129cf61

244429512-dae79bae-e572-40ce-84c6-560c90a6378b

244429604-7eaf25bd-b72d-4b8c-961e-f695a706ad2a

### CODE REVIEW

File: schema.sql
Comment (Alusp) [inline schema.sql:1]: - [ ] Good job so Far, :clap: , However, according to the project-specific requirement, `Everything that creates or change the tables structure should be saved in the schema.sql file.`, which means  your CREATE, DELETE,   statements  should be in `schema.sql`. Please resolve that. :slightly_smiling_face:

File: queries.sql
Comment (Alusp) [inline queries.sql:1]: - [ ] Good job so Far, :clap: , However, according to the project-specific requirement, `Everything that is a query that helps to analyze data should be saved in the queries.sql file.`, which means only your `SELECT`  statements  should be in `queries.sql`. Please resolve that. :slightly_smiling_face:

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @Mike47ip, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 close

## Highlights

- [ ] Completed half of the Tasks :clap: 
- [ ] Professional Readme File Used :clap: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql
Comment (Mike47ip) [reply schema.sql:1]: Some DATA.sql and QUERY.sql  functions sometime share borrowed clauses like INSERT AND SET so its difficult separately these files entirely.

**Review by Ibizugbe — APPROVED** [review]
Hi @Mike47ip 👋🏼👋🏼👋🏼, 

approved


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅✅

- [x] Query commands have been moved to the `query.sql` file👍🏼.
- [x] Create table commands have been moved to the `schema.sql` file👍🏼.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**